docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_DLRM.md

```diff
@@ -4,7 +4,9 @@
 
 These instructions are applicable only to the DLRM converted to the ONNX* file format from the [facebookresearch/dlrm model](https://github.com/facebookresearch/dlrm).
 
-**Step 1**. Save trained Pytorch* model to ONNX* format. If you train the model using the [script provided in model repository](https://github.com/facebookresearch/dlrm/blob/master/dlrm_s_pytorch.py), just add the `--save-onnx` flag to the command line parameters and you'll get the `dlrm_s_pytorch.onnx` file containing the model serialized in ONNX* format.
```

Comment:
Add the word "DEPRECATED" to the first line of the file as well

Reply:
done